lib/plugins/create/templates/aws-alexa-typescript/serverless.yml

```diff
@@ -1,5 +1,7 @@
 service:
   name: aws-alexa-typescript
+#app: your-app-name
+#tenant: your-tenant-name
```

Comment:
Was that change intentional?

Reply:
And if so, are all templates updated? What about Azure and Google?

Reply:
not in this pr. oops.